src/core/annotation.js

```diff
@@ -1517,6 +1590,24 @@ class TextWidgetAnnotation extends WidgetAnnotation {
 
     return chunks;
   }
+
+  async getAnnotationObject() {
+    return {
+      id: this.data.id,
+      value: this.data.fieldValue,
+      valueAsString: this.data.fieldValue.toString(),
```

Comment:
Is `this.data.fieldValue` guaranteed to be of a type that has a `toString()` method, because otherwise there'll be errors!?